python/ray/serve/_private/router.py

```diff
@@ -358,11 +596,14 @@ async def assign_replica(self, query: Query) -> ray.ObjectRef:
         and only send a query to available replicas (determined by the
         max_concurrent_quries value.)
         """
+        if query.metadata.is_streaming:
+            raise NotImplementedError("Streaming requires new routing to be enabled.")
```

Comment:
Could you explain how to enable new routing in this error message?

Reply:
This is really an internal assertion, should never be hit because we flip this flag on if streaming is enabled

Reply:
Sounds good.